server/controllers/__tests__/datesControllersIntegration.test.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios')
+jest.mock('axios')
+
+
+
+describe('checking values and retrieved from route call', () => {
+    test.skip('should return date', () => {
+        const mockDate = {"status": "success", "data": [{"value":"05/12/2021","type":"date"}, {"value": "05/13/2021", "type":"date"}]}
+        axios.get.mockResolvedValue(mockDate)
+        const { getDates } = require('../__mocks__/getDates')
```

Comment:
Can we move this to the top of the file?

Reply:
which line 10?

Reply:
Test fails when I move it.

Reply:
Removing that portion where I called the method stopped this error. Also our discussion regarding this test and integration tests in general is duly noted.